### DISCUSSION TITLE
Are `check` and `ci` is different?

### DISCUSSION THREAD

**Discussion by ikanago** [biomejs/biome#159] [Q&A] [answered]
We have options to view checks by `biome check PATH` (without `--apply`) or `biome ci PATH`.
Are there any differences between them such as `ci` has better performance or something?

**Comment by ematipico** [comment, marked as answer]
At the moment they might seem very similar because they are, but with time they will become different.

As for today, with `check` you can apply changes (formatting, lint code actions, etc.) to your project. With `ci` you can only go in read mode.

In the future `ci` will be able to print GitHub diagnostics, for example.

**Reply by markscamilleri** [reply]
Sorry to hop on here! I'm configuring biome for a project and this question popped into my head - and the docs were not clear. Sounds like we need to use `biome ci` for CI/CD builds for future features. 

Many tools are able to auto-detect and run in read-only mode - I believe this is done by detecting `CI` in the environment variables, or a similar one. Is this something that is planned?

**Comment by ikanago** [comment]
It can be helpful to show a result of `biome ci` in PR review comment. Thank you for your answer!

**Comment by PaulRBerg** [comment]
It would also be helpful to explain the difference between `biome check` and `biome ci` in the [docs for `ci`](https://biomejs.dev/reference/cli/#biome-ci). I had to search and find this discussion because this wasn't clear from the docs.
